src/client/components/chart-data-buttons/chartbar-buttons/chart-data-button.component.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import './chart-data-button.style.css';
+import PropTypes from 'prop-types';
+import Button from '../../button/button.component';
+
+const ChartbarMenu = ({ buttonLabel }) => {
+  const [selectedButton, setSelectedButton] = useState(buttonLabel.buttonWeek);
+  return (
+    <div className="chartbar-button-wrapper">
+      {buttonLabel.map((obj) => {
+        return (
+          <Button
+            type="toggle"
+            size="large"
+            onClick={(e) => {
+              e.preventDefault();
+              setSelectedButton(obj.buttonId);
+            }}
```

Comment:
Also, I don't think you need to `preventDefault` here.

If so, you could make it a one-liner 🙌